llmfoundry/callbacks/generate_callback.py

```diff
@@ -1,119 +1,24 @@
 # Copyright 2022 MosaicML LLM Foundry authors
 # SPDX-License-Identifier: Apache-2.0
 
-"""Periodically log generations to wandb from a set of prompts."""
-from typing import Any, List, Union, cast
+"""Deprecated Generate callback. Please use composer.callbacks.Generate instead."""
```

Comment:
I think most, if not all, users will use this callback via the yaml config. Is that correct @dakinggg ? If so, I think we should just remove this class altogether.

Reply:
Discussed offline with Daniel. We should keep it but will remove in a later release.